aequilibrae/project/database_specification/triggers/network_triggers.sql

```diff
@@ -221,7 +223,13 @@ create trigger deleted_link after delete on links
     end;
 --#
 -- when moving OR creating a link, don't allow it to duplicate an existing link.
```

Comment:
these comments don't line up with the code?

Reply:
I had ignored the comments when updating the triggers. Good catch. Fixed